components/LaunchPad/Token/CreateToken.tsx

```diff
@@ -232,9 +228,19 @@ export const CreateToken = () => {
       lastToastId = toast.loading(msg, { duration: 30 * 60 * 1000 }) // Awaiting signature, or awaiting confirmations
     }
     // toast.loading('AWAITING SIGNATURE', { duration: 30 * 60 * 1000 })
-    createAsset(formData, algodex.algod, activeWallet, notifier).then(
-      (asset) => (lastToastId = toast.success('sucess'))
-    )
+    createAsset(formData, algodex.algod, activeWallet, notifier)
+      .then((asset) => {
+        setLoading(false)
+        lastToastId = toast.success('sucess')
+        setLoading(false)
```

Comment:
Looks pretty good to me was just wondering why we set loading to false twice if `createAsset()` was successful

Reply:
That is a mistake, will remove it